datadog_checks_dev/datadog_checks/dev/stubs/aggregator.py

```diff
@@ -9,8 +9,7 @@
 
 from datadog_checks.base.stubs.common import HistogramBucketStub, MetricStub, ServiceCheckStub
 from datadog_checks.base.stubs.similar import build_similar_elements_msg
-
-from ..utils.common import ensure_unicode, to_native_string
+from datadog_checks.base.utils.common import ensure_unicode, to_native_string
```

Comment:
`from .stubs ...` no need to import from base no?

Reply:
Actually agreed with @ofek, looks like there are lots of opportunities for updating imports from `base` to `dev` in the moved files.